exercises/APE7/0702.tex

```diff
@@ -0,0 +1,89 @@
+    
+    Let's analyse now the payoff of the two scenarios available for player 1 at the information state 1.1: 
+    \begin{align*}
+        U_1(a_1|1.1) &= 1 \cdot 2 + 0 \cdot 8 = 2\\
+        U_1(b_1|1.1)& = \frac{1}{4} \cdot 9 + \frac{3}{4} \cdot 1 = 3
+    \end{align*}
+    hence $U_1(b_1|1.1)>U_1(a_1|1.1) $. It means that the player one will always play $b_1$  and never $a_1$. We deduce that $\alpha $ must be equal to 0, which is a contradiction with $\alpha \geq \frac{1}{6}$. This move is thus not in the sequential equilibria.
+    \item $D_2=\{b_2\}$: as player two will play $b_2$, $\beta$ must be equal to 0 (hence $a_2$ will never be played). Again, we need to have $b_2$ at least as preferable as $a_2$ : $$-\lambda \leq -6\lambda + 4 (1-\lambda)$$
+    and we find that $\alpha \leq \frac{1}{6}$ (by using the relation of $\lambda$ and $\alpha$). 
+    
+   Again, let's look at the payoff of the two scenarios available for player 1 at the information state 1.1: 
+   \begin{align*}
+       U_1(a_1|1.1) &= 0 \cdot 2 + 1 \cdot 8 = 8\\
+       U_1(b_1|1.1) &= \frac{1}{4} \cdot 1 + \frac{3}{4} \cdot 1 = 1
+   \end{align*} hence $U_1(a_1|1.1)>U_1(a_1|1.1) $. It means that the player one will always play $a_1$ and never $b_1$. We deduce that $\alpha$ must be equal to 1, which is a contradiction with $\alpha \leq \frac{1}{6}$. This sequential move is thus not in the sequential equilibria.
+    \item $D_2=\{a_2,b_2\}$ : we will now deal with the case where player two plays with a randomized strategy. This time we will thus have 
+    $$-\lambda = -6\lambda + 4 (1-\lambda)$$ so that player two does not prefer either $a_2$ or $b_2$. We obtain $\alpha = \frac{1}{6}$ (by using the relation of $\lambda$ and $\alpha$). 
+    
+    We need to check now for which values of $\beta$ player one is indifferent between playing $a_1$ and $b_1$. We have 
+    \begin{align*}
+        U_1(a_1|1.1)&= 2\beta + 8 (1-\beta) = 8 - 6 \beta\\
+        U_1(b_1|1.1)&=\frac{1}{4} (9\beta + (1-\beta) ) + \frac{3}{4} \cdot 1 = 1 + 2\beta
+    \end{align*} 
+    Let's equal those two expressions and we find that $\beta = \frac{7}{8}$. 
+    
+    We have thus found the final part of the sequential equilibria: for this equilibria, $\alpha=\frac{1}{6}$ (information state 1.1), $\beta =\frac{7}{8}$ (information state 2.2).
+\end{itemize}
+    
+The equilibria  ($\tau$ is the behavioural strategy and $\pi$ is the belief vector) is :
+\begin{align*}
+        [\tau, \pi ]&= \left[ \left(\tau_{1.1},\tau_{2.2},\tau_{2.3} \right), \left(\pi_{1.1}, \pi_{2.2} , \pi_{2.3}\right) \right]\\
+        &=\left[ \left(\frac{1}{6} a_1 + \frac{5}{6} b_1, \frac{7}{8} a_2 +\frac{1}{8} b_2, d_2\right) , \left( 1, (\frac{4}{9};\frac{5}{9}),1\right)\right]
```

Comment:
Write 4/9 [2.2.a_1] + 5/9 [2.2.b_1], it is not clear which one is the first and which one the second